src/__screenshot_tests__/form-fields-screenshot-test.tsx

```diff
@@ -27,6 +27,27 @@ test.each(TESTABLE_DEVICES)('Default textfield appears properly on %s', async (d
     expect(image).toMatchImageSnapshot();
 });
 
+test('Textfield collapses its label when autofill fills out the form', async () => {
+    await openStoryPage({
+        id: 'components-input-fields--variants',
+        device: 'DESKTOP',
+    });
+
+    const fieldWrapper = await screen.findByTestId('normal-field');
+    const image = await fieldWrapper.screenshot();
+
+    expect(image).toMatchImageSnapshot();
+
+    await page.evaluate(() => {
```

Comment:
since autofill can't be performed on puppeteer, I simulate it this way: in case the fix isn't applied, this simulation makes the test fail

Reply:
please, include a comment here explaining we are simulating an autofill